azurerm/resource_arm_bot_channel_msteams_test.go

```diff
@@ -0,0 +1,191 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/botservice/mgmt/2018-07-12/botservice"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func testAccAzureRMBotChannelMsTeams_basic(t *testing.T) {
+	// if ok := skipMsTeamsChannel(); ok {
```

Comment:
This can be removed since an MsTeams channel doesn't need any credentials to be configured